lib/adapters/autocomplete-adapter.js

```diff
@@ -101,6 +101,10 @@ export default class AutocompleteAdapter {
       description: item.documentation,
       descriptionMarkdown: item.documentation,
     };
+    if (item.insertTextFormat == 2) {
+      sugg.snippet = item.insertText;
+    };
+    return sugg;
```

Comment:
Thanks for this! Could we maybe go with just a 1-line change version of having the snippet added, e.g.

snippet: item.insertTextFormat == 2 ? item.insertText : undefined